models/item_for_rent.js

```diff
@@ -3,9 +3,15 @@ var Schema = mongoose.Schema;
 
 var item_for_rent_schema = new Schema({
     name: { type: String, unique: true },
+    type : String,
     description: String,
     quantity: Number,
-    price: Number,
+    Price_per_day: Number,
```

Comment:
http://stackoverflow.com/questions/9868323/is-there-a-convention-to-name-collection-in-mongo-db

Il faut faire attention avec les cases dans les schemas. Il mesemble que mongoose traduit tous les nom en lowercase, donc je crois qu'il serait pertinent d'utiliser les mêmes noms qui vont se retrouver dans la db au final. Sinon je crois qu'on devrais surtout faire attention avec le lengths des propriétés... 
See: http://docs.mongodb.org/manual/faq/developers/#faq-small-documents